nab/detectors/gaussian/windowedGaussian_detector.py

```diff
@@ -53,8 +76,8 @@ def handleRecord(self, inputData):
     anomalyScore = 0.0
     inputValue = inputData["value"]
     if len(self.windowData) > 0:
-      anomalyScore = 1 - anomaly_likelihood.normalProbability(
-        inputValue, {"mean": self.mean,"stdev": self.std})
+      anomalyScore = 1 - normalProbability(
+          inputValue, {"mean": self.mean, "stdev": self.std})
```

Comment:
No need to pass in a dict - just pass in self.mean and self.std and redefine normalProbability() accordingly.